apps/sms/views/inbox/js/startup.js

```diff
@@ -39,6 +40,7 @@
       ConversationClient.init(App.instanceId);
       MessageManager.init();
       Navigation.init();
+      Drafts.init();
```

Comment:
Yeah, I think we discussed this some time ago - it can be quite tricky especially for the conversation drafts - both instances can have different drafts and we don't want to discard one of them if another has been changed, but if one instance doesn't have it's own draft maybe we can do it...